Makefile

```diff
@@ -72,22 +72,26 @@ TMP_FILES := bpf/kmesh/bpf2go/bpf2go.go \
 	bpf/kmesh/ads/include/config.h \
 	bpf/include/bpf_helper_defs_ext.h \
 
-.PHONY: all
-all:
+.PHONY: all data daemon
+all: data daemon
+
+data:
```

Comment:
The name is confusing, this seems building bpf-prog,  how about `bpf-prog`

Reply:
not only bpf-prog, also deserialization.so kmesh.ko and bpf-prog

Reply:
I think any name is better and concrete than data. 

How about separating build ko and bpf